### PR TITLE
CNV-22364: Add note to "Expose port as service" section

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.13

Issue:  [CNV-22364](https://issues.redhat.com/browse/CNV-22364)

Link to docs preview: https://56089--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-creating-service-vm.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by aspauldi** [comment]
@phoracek, This PR is ready for dev review; thanks!

**Review by phoracek — CHANGES_REQUESTED** [review]

File: modules/virt-about-services.adoc [hunk 1]
```diff
@@ -6,9 +6,17 @@
...
 
-A Kubernetes _service_ is an abstract way to expose an application running on a set of pods as a network service. Services allow your applications to receive traffic. Services can be exposed in different ways by specifying a `spec.type` in the `Service` object:
+A Kubernetes _service_ exposes network access for clients to an application running on a set of pods. Services offer abstraction, load balancing, and, in the case of NodePort and LoadBalancer, exposure to the outside world. 
 
-ClusterIP:: Exposes the service on an internal IP address within the cluster. `ClusterIP` is the default service `type`.
+
+[NOTE]
+====
+Creating a dedicated service, not shared by others, is recommended for a high-traffic connection. 
+====
```
[the remark this answers is not in the document]
Comment (aspauldi) [reply modules/virt-about-services.adoc:15]: Hi Petr, That line came from info in the Jira story description.

[the remark this answers is not in the document]
Comment (phoracek) [reply modules/virt-about-services.adoc:15]: I see. It's true that this provides a better performance than connecting via SSH though a proxy. However, out of the context this note sounds that services add performance benefit on their own, which is not the case.

I would suggest we drop this note and rely on https://docs.openshift.com/container-platform/4.12/virt/virtual_machines/virt-accessing-vm-consoles.html#virt-accessing-vmi-ssh_virt-accessing-vm-consoles which already describes downsides of `virtctl ssh`. I would treat Services as the default, they are not particularly slow (like virtctl ssh) nor fast (like SR-IOV).

**Comment by aspauldi** [comment]
@phoracek 

Can you take another look? I removed the NOTE. Thanks!

**Review by phoracek — APPROVED** [review]
Looks good, thanks!

**Comment by aspauldi** [comment]
Thanks, Petr!

**Comment by aspauldi** [comment]
@yossisegev , Hello! Can you please take a look at this update? Thanks!

**Comment by yossisegev** [comment]
LGTM

**Comment by aspauldi** [comment]
Thanks, @yossisegev !

**Review by kowen-rh — APPROVED** [review]
LGTM! :+1:

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

**Pull request merged**